deploy/crds/org.eclipse.che_checlusterbackups_crd.yaml

```diff
@@ -0,0 +1,214 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: checlusterbackups.org.eclipse.che
+spec:
+  group: org.eclipse.che
+  names:
+    kind: CheClusterBackup
+    listKind: CheClusterBackupList
+    plural: checlusterbackups
+    singular: checlusterbackup
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: CheClusterBackup is the Schema for the checlusterbackups API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: CheClusterBackupSpec defines the desired state of CheClusterBackup
+          properties:
+            autoconfigureRestBackupServer:
+              description: Automatically setup container with REST backup server and
+                use the server in this configuration. Note, this will overwrite existing
+                configuration.
+              type: boolean
+            serverType:
+              description: If more than one backup server configured, should specify
+                which one to use. Allowed values are fields names form BackupServers
```

Comment:
I think `BackupServers` struct is a reference to go struct, but with CRD it's not really clear which field you refer to. Please try to rewrite in a way that would work for go and Json Schema.